include/RAJA/util/resource.hpp

```diff
@@ -36,7 +37,7 @@ namespace RAJA
 
   template<typename e>
   struct get_resource{
-    using type = camp::resources::Host;
+    using type = Host;
```

Comment:
Could we keep the full namespace? I ran into namespace issues when building a large application with RAJA.

Reply:
Fixed 👍